src/components/Input/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
I would wait with this implementation of this component until we have a new form lib.
we should create a wrapper component for all types of form components with a label and a subtitle and the field.
but if it is needed now just add it.

Reply:
We're going to need the `asc-ui` implementation anyway, regardless of which lib we choose.